libr/debug/p/debug_native.c

```diff
@@ -1786,7 +1704,9 @@ static RList *xnu_desc_list (int pid) {
 
 #if __WINDOWS__
 static RList *win_desc_list (int pid) {
-	RDebugDesc *desc;
+	return NULL;
+	// Disabling this for now
```

Comment:
why disabling it. thats a regression. we want this

Reply:
This is actually a mistake on my part leaving it commented out as it was only meant to be commented out while I was working on this pr.